ivc/src/ivc/columns.rs

```diff
@@ -212,6 +218,9 @@ pub type IVCPoseidonColumn = PoseidonColumn<IVC_POSEIDON_STATE_SIZE, IVC_POSEIDO
 // TODO: Can we pass just one coordinate and sign (x, sign) instead of (x,y) for hashing?
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
 pub enum IVCColumn {
+    /// Selector for blocks. Inner usize is ∈ [0,#blocks).
+    BlockSel(usize),
```

Comment:
Just to make sure, this is not a row selector (as in Kimchi) but rather a selector for a chunk of columns within the same row, no?

Reply:
Oh I see then yes, like having the same Kimchi selector in consecutive rows.